eng/testing/RunnerTemplate.sh

```diff
@@ -1,207 +0,0 @@
-﻿#!/usr/bin/env bash
-
-usage()
-{
-  echo "Usage: RunTests.sh {-r|--runtime-path} <runtime-path> [{--rsp-file} <rsp-file>]"
-  echo ""
-  echo "Parameters:"
-  echo "--runtime-path           (Mandatory) Testhost containing the test runtime used during test execution (short: -r)"
-  echo "--rsp-file               RSP file to pass in additional arguments"
-  echo "--help                   Print help and exit (short: -h)"
-}
-
-EXECUTION_DIR=$(dirname "$0")
-RUNTIME_PATH=''
-RSP_FILE=''
-
-while [[ $# > 0 ]]; do
-  opt="$(echo "${1}" | awk '{print tolower($0)}')"
-  case "$opt" in
-    --help|-h)
-      usage
-      exit 0
-      ;;
-    --runtime-path|-r)
-      RUNTIME_PATH=$2
-      shift
-      ;;
-    --rsp-file)
-      RSP_FILE=\@$2
-      shift
-      ;;
-    *)
-      echo "Invalid argument: $1"
-      usage
-      exit -1
-      ;;
-  esac
-
-  shift
-done
-
-if [ "$RUNTIME_PATH" == "" ]; then
-  echo "error: -r|--runtime-path argument is required."
-  usage
-  exit -1
-fi
-
-# Don't use a globally installed SDK.
-export DOTNET_MULTILEVEL_LOOKUP=0
-
-exitcode_list[0]="Exited Successfully"
```

Comment:
I found it useful sometimes for it to translate the signal like this. Is it still relevant?

Reply:
That would probably be a nice feature for VSTest.

Reply:
Can we keep it here until `vstest` supports it or has such a feature?

Reply:
We don't use RunnerTemplates and I'm unsure if we get access to the testhost's ExitCode. I guess easiest would be to just send a PR to vstest, that shouldn't take long. will do.

Reply:
Never mind, I didn't know we were removing this file. I thought it was removing some lines 😄